sushi-config.yaml

```diff
@@ -72,6 +73,8 @@ pages:
     title: Security, Privacy, and Consent
   signatures.md:
     title: Signatures
+  search_parameters.md:
+    title: Search Parameters Required By This Implementation Guide
```

Comment:
Should we change this title from "Search Parameters Required..." to "Search Parameters Supported..."?

Reply:
nvm. this is explained.